object/check.go

```diff
@@ -33,6 +34,11 @@ const (
 	DefaultFailedSigninFrozenTime = 15
 )
 
+var (
+	IPv4CIDRRegex = regexp.MustCompile(`^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\/([0-9]|[1-2][0-9]|3[0-2]))$/`)
```

Comment:
you can use `net.ParseCIDR` to check instead of use regex

Reply:
yes, i have changed it just now